RELEASING.md

```diff
@@ -108,10 +108,5 @@ From the project's root directory, run the following to export the environment v
 to your local `~/.m2` repository.
 ''
```

Comment:
Can you remove the extra quote.

Reply:
added to https://github.com/grafana/grafana-opentelemetry-starter/pull/30 - which is also ready for review :smile: